service/web_api.py

```diff
@@ -195,6 +195,33 @@ def check_model_exist():
     return jsonify({"code": 0, "message": "success", "exists": result_list})
 
 
+@app.route("/api/checkURLExists", methods=["POST"])
```

Comment:
I think this should be a "GET" request, the other method as well. We then would need to figure out, if we provide the address as path or query parameter. See this https://stackoverflow.com/questions/30967822/when-do-i-use-path-parameters-vs-query-parameters-in-a-restful-api for a fruitful distinction on how to use it.

Reply:
that is true. Changed the requests to GET for our implementation, query should be the obvious choice here, right?. How about the isModelGated requests with complex parameters? A query doesn't seem to do the job here...